code/modules/cargo/packs.dm

```diff
@@ -1222,6 +1222,19 @@
 					/obj/item/roller)
 	crate_name = "surgical supplies crate"
 
+/datum/supply_pack/medical/lobotomy
```

Comment:
I'm not sure I'm a fan of locking lobotomies without the psychologist behind cargo. Are you sure you don't want to put the tools behind the fabricator? Are you sure it's necessary to remove the normal lobotomy in favor of this, anyway?

Reply:
I think it is uncommon enough for cargo to be a fine way of getting it without it being something every doctor can add to their belt.